mutiny-core/src/nodemanager.rs

```diff
@@ -1765,17 +1765,34 @@ impl<S: MutinyStorage> NodeManager<S> {
 
         match channel_opt {
             Some((node, channel)) => {
-                node.channel_manager
-                    .close_channel(&channel.channel_id, &channel.counterparty.node_id)
-                    .map_err(|e| {
-                        log_error!(
-                            self.logger,
-                            "had an error closing channel {} with node {} : {e:?}",
-                            &channel.channel_id.to_hex(),
-                            &channel.counterparty.node_id.to_hex()
-                        );
-                        MutinyError::ChannelClosingFailed
-                    })?;
+                if force {
+                    node.channel_manager
+                        .force_close_broadcasting_latest_txn(
```

Comment:
THe issue where the channel opening will never confirm, should that be by force closing without broadcasting latest txn?

Reply:
hmm true, maybe add an `abandon_channel` function that does that

Reply:
added

Reply:
What do you think of having `force` and `abandon` as a parameter here? The code is pretty similar.